### ISSUE TITLE
[Android] Unable to compile RN project

### ISSUE THREAD

**Issue by Vol0kin** [lottie-react-native/lottie-react-native#873]
### Description

The latest version of Lottie introduces some compilation errors on Android devices caused by not using final variables in inner classes.

### Steps to Reproduce

1. Upgrade or install latest version of `react-native-lottie`  (currently v5.1.1).
2. Start Metro server.
3. Run Android project.

**Expected behavior:**

Project should run on real or virtual Android device.

**Actual behavior:**

The following error message is shown:

```
BUILD FAILED in 13s

error Failed to install the app. Make sure you have the Android development environment set up: https://reactnative.dev/docs/environment-setup.
Error: Command failed: ./gradlew app:installDebug -PreactNativeDevServerPort=8081
Note: /home/volokin/Repositories/Work/SampleProject/node_modules/@sentry/react-native/android/src/main/java/io/sentry/RNSentryModule.java uses or overrides a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
/home/volokin/Repositories/SampleProject/node_modules/lottie-react-native/src/android/src/main/java/com/airbnb/android/react/lottie/LottieAnimationViewManager.java:220: error: local variable urlString is accessed from within inner class; needs to be declared final
              BufferedReader in = new BufferedReader(new InputStreamReader(new URL(urlString).openStream()));
                                                                                   ^
/home/volokin/Repositories/Work/SampleProject/node_modules/lottie-react-native/src/android/src/main/java/com/airbnb/android/react/lottie/LottieAnimationViewManager.java:233: error: local variable view is accessed from within inner class; needs to be declared final
                  getOrCreatePropertyManager(view).setAnimationJson(js);
                                             ^
/home/volokin/Repositories/Work/SampleProjectr/node_modules/lottie-react-native/src/android/src/main/java/com/airbnb/android/react/lottie/LottieAnimationViewManager.java:233: error: local variable js is accessed from within inner class; needs to be declared final
                  getOrCreatePropertyManager(view).setAnimationJson(js);
                                                                    ^
/home/volokin/Repositories/SampleProject/node_modules/lottie-react-native/src/android/src/main/java/com/airbnb/android/react/lottie/LottieAnimationViewManager.java:234: error: local variable view is accessed from within inner class; needs to be declared final
                  getOrCreatePropertyManager(view).commitChanges();
```

### Versions

v5.1.1

**Comment by namjaehyun-me** [comment]
Change version to 5.0.1 :)

**Comment by epkim** [comment]
If you install the existing ^5.1.0 version from package.json, the 5.1.0 version will be installed. Change the ^5.1.0 version to 5.0.1 and reinstall.

**Comment by emilioicai** [comment]
thanks @Vol0kin this is fixed on v5.1.2
